JuliaGPU/session2/Feyaz/qn1.jl

```diff
@@ -0,0 +1,30 @@
+#Try using arr as the 10 digit array, if you're impatient.
```

Comment:
It's not about patience, it's about efficiency! The question forced the size of the array to be 2^20 for a reason. Insertion sort is inefficient and hence can take more than 2 hours to sort such large array. So, you will need to implement a more efficient sorting algorithm (like merge sort or quick sort) so that you can benchmark it.

Reply:
I'm having some difficulty in implementing merge sort. On my system insertion sort ran in under 5 minutes. Could you check and tell if I have to redo this question?

Reply:
It took 7 minutes in my system (I am a bit baffled it ran so fast! Expected it to take an hour at least).   
Still, 7 minutes (or 5, in your system) is too long compared to Julia's sort which takes less than a second. So it is better if you implement a more efficient sorting algorithm. You can share your issues with merge sort implementation here.